File: VeamIt-project/iOS/veam00000000/veam00000000/TemplateWeb.m
//
//  TemplateWeb.m
//  veam00000000
//
//  Created by veam on 6/4/14.
//  Copyright (c) 2014 veam. All rights reserved.
//

#import "TemplateWeb.h"

@implementation TemplateWeb

@synthesize templateWebId ;
@synthesize title ;

- (id)initWithAttributes:(NSDictionary *)attributeDict
{
    self = [super init] ;
    
    [self setTemplateWebId:[attributeDict objectForKey:@"id"]] ;
    [self setTitle:[attributeDict objectForKey:@"t"]] ;
    
    return self ;
}

@end

File: VeamIt-project/Android/veam31000287/veam31000287/src/main/java/co/veam/veam31000287/TemplateWeb.java
package co.veam.veam31000287;

import org.xml.sax.Attributes;

/**
 * Created by veam on 10/27/16.
 */
public class TemplateWeb {
    private String templateWebId ;
    private String title ;

    public TemplateWeb() {

    }

    public String getTemplateWebId() {
        return templateWebId;
    }

    public void setTemplateWebId(String templateWebId) {
        this.templateWebId = templateWebId;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public TemplateWeb(Attributes attributes) {
        this.setTemplateWebId(attributes.getValue("id")) ;
        this.setTitle(attributes.getValue("t")) ;
    }
}
